### ISSUE TITLE
Site should include Rosters and Intros

### ISSUE THREAD

**Issue by ninomario** [superwarcraftbros/website#4]
Roster I assume is a given, but would be good to do an **intro for GM's / Officers** of the guild so we are approachable.

This could be in the form of Photos or Videos

**Comment by Spoons-Nick** [comment]
Yeah that's a nice idea, could maybe integrate something like a wall of fame or something? Just to give shout outs to players who go above and beyond. "employee of the week" kind of thing
